src/resources/css/parts-kit.css

```diff
@@ -65,7 +65,53 @@
 .parts-kit__sidebar .flex {
     display: flex;
 }
+/* Search */
+.parts-kit__search {
+    padding: 0 18px;
+    margin-bottom: 8px;
+    position: relative;
```

Comment:
nit: typically our stylelint rule also enforces alphabetizing properties. such a minor detail

Reply:
Did we ever figure out how to make prettier do this format on save?